lib/nexus/include/nexus/unit.hpp

```diff
@@ -57,13 +59,19 @@ namespace nex {
             // transmissibility,
             viscosity,
             volume,
-            water_cut
+            water_cut,
+            measure_count
```

Comment:
I would guess count and identity - i.e. no dimensionless number - could be the same?

Reply:
This did confuse me somewhat as well actually. That entry wasn't actually meant to be a measure, but the count of measures in the enum.

Reply:
I've renamed it to make this clearer